src/controllers/controller.bar.js

```diff
@@ -453,8 +473,14 @@ module.exports = function(Chart) {
 
 			helpers.canvas.clipArea(chart.ctx, chart.chartArea);
 
+			//  float-bar support, if y arguments are array function will use bottom value as bar start point
 			for (; i < ilen; ++i) {
-				if (!isNaN(scale.getRightValue(dataset.data[i]))) {
+				var yValue = dataset.data[i];
+				if (helpers.isArray(yValue)) {
```

Comment:
I think we should check `min !== max` here like we do elsewhere instead of `isArray`